src/trace/Component.ts

```diff
@@ -28,6 +28,9 @@ export class Component {
   static readonly RABBITMQ_PRODUCER = new Component(52);
   static readonly RABBITMQ_CONSUMER = new Component(53);
   static readonly AZURE_HTTPTRIGGER = new Component(111);
+  static readonly AWSLAMBDA_FUNCTION = new Component(124);
+  static readonly AWSLAMBDA_GATEWAYAPIHTTP = new Component(125);
+  static readonly AWSLAMBDA_GATEWAYAPIREST = new Component(126);
```

Comment:
Can you find some proper IDs for these plugins?

https://github.com/apache/skywalking/blob/68e40eb9fc0a891a55fe927d49b267d0fd80000c/oap-server/server-starter/src/main/resources/component-libraries.yml#L459

Reply:
Yes they will be fixed, it may be a little bit though as I have several things at the moment. The IDs are already found, just haven't sent them up to the main SW repo, will do when this is fixed.

Reply:
> Yes they will be fixed, it may be a little bit though as I have several things at the moment. The IDs are already found, just haven't sent them up to the main SW repo, will do when this is fixed.

These IDs 124~126 seems to be in the reserved range for other language, not NodeJS, so better choose those inside the range for NodeJS, (e.g. `400` ~ `5000`)

https://github.com/apache/skywalking/blob/68e40eb9fc0a891a55fe927d49b267d0fd80000c/oap-server/server-starter/src/main/resources/component-libraries.yml#L459-L460

Reply:
That is the same place we put the Azure Functions ID. Also, these are not limited to NodeJS, you can run lots of languages on Lambda with these triggers.

Reply:
Ok, so do I push the IDs I have to main repo or do you want change?

Reply:
> Ok, so do I push the IDs I have to main repo or do you want change?

Please do that, using 124~126 is good to me